docs/guide/1.x/setup.md

```diff
@@ -0,0 +1,138 @@
+# Setting up
+Spek is implemented as a [JUnit Platform](https://junit.org/junit5) TestEngine. 
+
+Include `org.jetbrains.spek:spek-api` and `org.jetbrains.spek:spek-junit-platform-engine` 
+in the test classpath. The former is needed during compilation, while the latter during runtime only.
+
+## Gradle
+Gradle `4.7` is recommended as it added a built-in support for JUnit Platform.
+
+```groovy
+// setup dependencies
+dependencies {
+    // some version of Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+
+    testImplementation ('org.jetbrains.spek:spek-api:1.x.x')  {
+        exclude group: 'org.jetbrains.kotlin'
+    }
+    testRuntimeOnly ('org.jetbrains.spek:spek-junit-platform-engine:1.x.x') {
+        exclude group: 'org.junit.platform'
+        exclude group: 'org.jetbrains.kotlin'
+    }
+    
+    // spek requires kotlin-reflect, can be omitted if already in the classpath
+    testRuntimeOnly "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
```

Comment:
Shouldn’t Spek declare it itself? It is required after all.

Reply:
See above.